lager_suite_test.go

```diff
@@ -1,12 +1,18 @@
 package lager_test
 
 import (
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"runtime"
 
 	"testing"
 )
 
+const MaxThreads = 100
+var _ = BeforeSuite(func() {
+	runtime.GOMAXPROCS(MaxThreads)
```

Comment:
Why is this necessary?

Reply:
If I remember correctly it was to enable more threads to reduce the runtime of the tests ... its been a while now.

Reply:
It also has been moved from [writer_sink_test.go](https://github.com/cloudfoundry/lager/pull/54/files/a053a8ef0561e8279b52a1dfe161ce61a92d6d3d#diff-f43d3d0989e10a86b7c8fa0eb6eb9ab4627cbbfcc8fc70a326794771033e5c18)